contracts/bridge/Bridge.sol

```diff
@@ -166,45 +171,46 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable {
         }
     }
 
-    // () requests transfer KLAY to msg.sender address on relative chain.
-    function () external payable {
+    // _requestKLAYTransfer requests transfer KLAY to _to on relative chain.
+    function _requestKLAYTransfer(address _to, uint256 _fee) internal {
         require(isRunning, "stopped bridge");
         require(msg.value > 0, "zero msg.value");
+        require(msg.value > _fee, "insufficient amount");
```

Comment:
It seemed that this line can cover the above line.

Reply:
Good. :)

Reply:
Now, let's pursue the right revert message for exception case. Thanks. :) I will consider next FeeLimit PR.